movies/src/main/java/comp.fundacionjala/movies/RegularMovie.java

```diff
@@ -0,0 +1,27 @@
+package comp.fundacionjala.movies;
+
+class RegularMovie extends Movie implements IMovie {
+
+    public static final double CHARGE_PER_RENTAL_DAY = 1.5;
+    public static final double INITIAL_CHARGE = 2;
+    public static final double THRESHOLD_DAYS = 2;
+    public static final int FREQUENT_RENTER_POINTS = 1;
```

Comment:
If you notice the following variables CHARGE_PER_RENTAL_DAY, INITIAL_CHARGE, THRESHOLD_DAYS and FREQUENT_RENTER_POINTS, they are repeated in the RegularMovie, NewReleaseMovie and ChildrenMovie classes.
You should abstract those variable in the Movie class